Makefile

```diff
@@ -223,3 +225,27 @@ endif
 dist : clean sparse.c sscan.c
 	find . -name '*~' -type f -exec rm {} \;
 	cd .. && tar --transform s/$(SRC_DIR)/pgsphere-$(PGSPHERE_VERSION)/ --exclude CVS --exclude .git -czf pgsphere-$(PGSPHERE_VERSION).tar.gz $(SRC_DIR) && cd -
+
+# To launch pgindent, set the PATH environment variable to the directories
```

Comment:
I propose changing this line to the following:

To use pgindent, set the PATH environment variable to include the directories